the ci results: https://github.com/Green-Sky/llama.cpp/actions/runs/4525140706

for reference, the compile error:
```
/home/runner/work/llama.cpp/llama.cpp/ggml.c:5839:20: error: ‘ne13’ undeclared (first use in this function); did you mean ‘nb13’?
 5839 |     assert(ne03 == ne13);
      |                    ^~~~
/home/runner/work/llama.cpp/llama.cpp/ggml.c:5840:12: error: ‘ne2’ undeclared (first use in this function); did you mean ‘nb2’?
 5840 |     assert(ne2  == ne12);
      |            ^~~
/home/runner/work/llama.cpp/llama.cpp/ggml.c:5841:12: error: ‘ne3’ undeclared (first use in this function); did you mean ‘nb3’?
 5841 |     assert(ne3  == ne13);
      |            ^~~
/home/runner/work/llama.cpp/llama.cpp/ggml.c:5844:12: error: ‘nb00’ undeclared (first use in this function); did you mean ‘nb0’?
 5844 |     assert(nb00 == sizeof(float));
      |            ^~~~
/home/runner/work/llama.cpp/llama.cpp/ggml.c:5852:12: error: ‘ne0’ undeclared (first use in this function); did you mean ‘nb0’?
 5852 |     assert(ne0 == ne01);
      |            ^~~
/home/runner/work/llama.cpp/llama.cpp/ggml.c:5853:12: error: ‘ne1’ undeclared (first use in this function); did you mean ‘nb1’?
 5853 |     assert(ne1 == ne11);
      |            ^~~
/home/runner/work/llama.cpp/llama.cpp/ggml.c:5809:15: warning: unused variable ‘ne10’ [-Wunused-variable]
 5809 |     const int ne10 = src1->ne[0];
      |               ^~~~
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -72,6 +72,8 @@ jobs:
     strategy:
       matrix:
         sanitizer: [ADDRESS, THREAD, UNDEFINED]
+        build_type: [Debug, Release]
+        accelerate: [ON, OFF]
 
     steps:
       - name: Clone
@@ -89,8 +91,8 @@ jobs:
         run: |
           mkdir build
           cd build
-          cmake .. -DLLAMA_SANITIZE_${{ matrix.sanitizer }}=ON
-          cmake --build . --config Release
+          cmake .. -DLLAMA_SANITIZE_${{ matrix.sanitizer }}=ON -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} -DLLAMA_ACCELERATE=${{ matrix.accelerate }}
+          cmake --build . --config ${{ matrix.build_type }}
 
       - name: Test
         id: cmake_test

